src/main/resources/changelog-master.xml

```diff
@@ -56,4 +56,5 @@
     <include file="changesets/changelog-consent-51.0.xml" relativeToChangelogFile="true"/>
     <include file="changesets/changelog-consent-52.0.xml" relativeToChangelogFile="true"/>
     <include file="changesets/changelog-consent-53.0.xml" relativeToChangelogFile="true"/>
+    <include file="changesets/changelog-consent-54.0.xml" relativeToChangelogFile="true"/>
```

Comment:
I grabbed 54 first, so you'll have to switch to 55

Reply:
Gotcha, thanks, will rebase.